build/log_shutdown.go

```diff
@@ -41,3 +43,15 @@ func (s *ShutdownLogger) Critical(v ...interface{}) {
 	s.Logger.Info("Sending request for shutdown")
 	s.shutdown()
 }
+
+// CriticalS writes a structured log with the given message and key-value pair
+// attributes with LevelCritical to the log.
```

Comment:
nit: add "It will then call the shutdown logger's shutdown function to prompt safe shutdown."